local/compose/ps.go

```diff
@@ -37,32 +38,49 @@ func (s *composeService) Ps(ctx context.Context, projectName string) ([]compose.
 		return nil, err
 	}
 
-	var summary []compose.ContainerSummary
-	for _, c := range containers {
-		var publishers []compose.PortPublisher
-		for _, p := range c.Ports {
-			var url string
-			if p.PublicPort != 0 {
-				url = fmt.Sprintf("%s:%d", p.IP, p.PublicPort)
+	summary := make([]compose.ContainerSummary, len(containers))
+	eg, ctx := errgroup.WithContext(ctx)
+	for i, c := range containers {
+		container := c
+		i := i
```

Comment:
I'll never get used to it... :)

Reply:
I tend to like it now, sort of a running gag